coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java

```diff
@@ -36,7 +36,7 @@
     
     public static final String BROWSER_PROTOCOL = "http";
     public static final String BROWSER_ADDRESS = "127.0.0.1";
-    public static final String PORTAL_ADDRESS = "kc-dev";
+    public static final String PORTAL_ADDRESS = "";
```

Comment:
did you run the full suite.  It doesn't take that long.  I'd suggest doing that.``` mvn clean install -Pitests```